topology/probes/k8s/pod.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/skydive-project/skydive/probe"
 	"github.com/skydive-project/skydive/topology/graph"
 
-	api "k8s.io/api/core/v1"
```

Comment:
using api namespace was much better for readability

Reply:
but this is not the standard way these packages are accessed from within the k8s code base ... either the default is used such as `v1` or `v1beta1` or if it conflicts with other packages then you append the parent directory such as `corev1` or `metav1beta1`

Reply:
so when they will switch (and the probe too) to a v2 of the API you want to rename all the code ?